packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -57,25 +56,20 @@ class PlatformViewsService {
     SystemChannels.platform_views.setMethodCallHandler(_onMethodCall);
   }
 
-  static PlatformViewsService _serviceInstance;
-
-  static PlatformViewsService get _instance {
-    _serviceInstance ??= PlatformViewsService._();
-    return _serviceInstance;
-  }
+  static final PlatformViewsService _instance = PlatformViewsService._();
```

Comment:
Should this be `late` to keep the laziness of the original code?

Reply:
Static variables and top level variables are already lazy in Dart so I think `static late final` is the same as `static final`.
@leafpetersen or @lrhn is it correct? 

If yes the analyzer could perhaps have an hint to prefer `static final` over `static late final`

Reply:
`static late final` with an initializer is the same as `static final` (and also when non-final).  Without an initializer they are different.  +1 for a hint/lint.  cc @bwilkerson